e2e/app/definition.go

```diff
@@ -424,8 +421,13 @@ func TestnetFromManifest(ctx context.Context, manifest types.Manifest, infd type
 // getOrGenKey gets (based on manifest) or creates a private key for the given node and type.
 func getOrGenKey(ctx context.Context, manifest types.Manifest, nodeName string, typ key.Type) (key.Key, error) {
 	addr, ok := manifest.Keys[nodeName][typ]
-	if !ok {
-		// No key in manifest, generate a new one.
+	if !ok { // No key in manifest
+		// Generate an insecure deterministic key for devnet
+		if manifest.Network == netconf.Devnet {
+			return key.GenerateInsecureDeterministic(manifest.Network, typ, nodeName), nil
```

Comment:
deterministic keys so e2e tests can verify and check validator updates which requires the same key